doodle_template.php

```diff
@@ -48,13 +48,16 @@
      <tr>
          <td class="rightalign"> 
            <?php
-		if ($template['printName'] == 'both'){
+		if (strcmp($template['printName'],'none')==0){
+                        echo $userData['editLinks']."anonyme";
```

Comment:
I have two suggestions.
1) Can you please change "anonyme" to "anonymous"?
2) Currently, this option will show "anonyme" for all the rows.  Instead, it would be nice if we can show the username (or fullname) in the row corresponding to the user and "anonymous" for everyone else.
So if there 100 votes, 99 rows will show up as "anonymous" and one row will show the username.